.github/workflows/deploy-vitest-ui.yaml

```
@@ -0,0 +1,45 @@
+
+      - name: Setup Pages
+        uses: actions/configure-pages@v5
+
+      - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+        with:
+          path: server/html
+
+      - name: Deploy to GitHub Pages
+        id: deployment
+        uses: actions/deploy-pages@v4
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)